extensions/smallrye-fault-tolerance/runtime/src/main/java/io/quarkus/smallrye/faulttolerance/runtime/HystrixInitializerStarter.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.smallrye.faulttolerance.runtime;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Observes;
+
+import io.quarkus.runtime.StartupEvent;
+import io.smallrye.faulttolerance.HystrixInitializer;
+
+@Dependent
+public class HystrixInitializerStarter {
+
+    /**
+     * This is a replacement for <code>io.smallrye.faulttolerance.HystrixInitializer.init(Object)</code> observer method which
+     * is vetoed because we don't want initialize Hystrix during static init.
+     * 
+     * @param event
+     * @param initializer
+     */
+    void startup(@Observes StartupEvent event, HystrixInitializer initializer) {
+        initializer.toString();
```

Comment:
Might be worth adding a note that we need to trigger any bean-specific method or `toString` in order to enforce bean creation. While it's obvious for us, I can see a lot of non-CDI folks staring at this suspiciously ;-)

Reply:
Ok, I can add a link to the "Eager Instantiation of Beans" section.

Reply:
Eh, sorry not a link but a comment.